source/blog/2016-04-20-scala-210-and-java7.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: Final Scala 2.10 and Java 7 release
+published: true
+post_author:
+  display_name: Kevin Oliver
+  twitter: kevino
+tags: Finagle, Finatra, Util, Scrooge, TwitterServer, Ostrich
+---
+
+The upcoming releases of our family of projects will be the final release for
+Scala 2.10 and Java 7. These should be Finagle 6.35.0, Finatra 2.2.0,
+Util 6.34.0, Scrooge 4.7.0, TwitterServer 1.20.0, and Ostrich 9.18.0.
```

Comment:
Should we mention that we're still going to publish 2.10 jars for Scrooge (right?) since scrooge-sbt-plugin needs that.